apps/meteor/client/sidebar/header/MatrixFederationSearch/FederatedRoomList.tsx

```diff
@@ -60,23 +60,25 @@ const FederatedRoomList: VFC<FederatedRoomListProps> = ({ serverName, roomName,
 
 	const flattenedData = data?.pages.flatMap((page) => page.rooms);
 	return (
-		<Box is='ul' overflow='hidden' height='356px' flexGrow={1} flexShrink={0}>
-			<Virtuoso
-				data={flattenedData || []}
-				computeItemKey={(index, room) => room?.id || index}
-				overscan={4}
-				components={{
-					// eslint-disable-next-line react/no-multi-comp
-					Footer: () => (isFetchingNextPage ? <Throbber /> : null),
-					Scroller: ScrollableContentWrapper,
-					EmptyPlaceholder: FederatedRoomListEmptyPlaceholder,
-				}}
-				endReached={isLoading || isFetchingNextPage ? () => undefined : () => fetchNextPage()}
-				itemContent={(_, room) => (
-					<FederatedRoomListItem onClickJoin={() => onClickJoin(room)} {...room} disabled={isLoadingMutation} key={room.id} />
-				)}
-			/>
-		</Box>
+		<Margins inline={-26}>
```

Comment:
Any reason this margin isn't inside the Box component?

Reply:
My only mistake wrong margin number